egressgw: drop traffic if no gateway is found (#24835)
cilium-team/cilium: Documentation/network/egress-gateway.rst

    In case multiple nodes are a match for the given set of labels, the
    first node in lexical ordering based on their name will be selected.

.. note::

>     In case there is no match for the given set of labels, Cilium will drop the
>     traffic that matches the destination CIDR(s).

Reviewer: Use present tense, edits for clarity
```suggestion
    If there is no match for the given set of labels, Cilium drops the
    traffic that matches the destination CIDR(s).
```
